schema/build-config-schema.js

```diff
@@ -531,6 +531,20 @@ for (const rule of rules) {
         },
       };
       break;
+    case "MD055":
+      scheme.properties = {
+        "style": {
+          "description": "Where border pipes are required",
```

Comment:
Why is this optional? My first thought is to have this rule require both - or else turn off the rule to require neither or not care. Supporting one-or-the-other is more code and more testing. Would anyone customize this?

Reply:
#93 includes an example with a border on one side. I'd also go with removing this complexity though.